daemon/src/projection.rs

```diff
@@ -1199,8 +1199,11 @@ impl xtra::Actor for Actor {
                 loop {
                     {
                         let span = info_span!("Update projection with latest quote");
+                        // TODO: subscribe to all the quotes
```

Comment:
🙃 Shall we already adapt the `State` to handle more than one `Quote`?

Reply:
That's done in #2615